marklogic-data-hub-central/ui/src/components/modeling/property-modal/property-modal.tsx

```diff
@@ -756,29 +757,28 @@ const PropertyModal: React.FC<Props> = (props) => {
 
   const modalFooter = <div className={props.editPropertyOptions.isEdit ? styles.editFooter : styles.addFooter}>
     { props.editPropertyOptions.isEdit &&
-      <Button type="link" onClick={async () => {
+      <HCButton variant="link" onClick={async () => {
         if (confirmType === ConfirmationType.Identifer) {
           await getEntityReferences();
         }
         toggleConfirmModal(true);
       }}>
         <FontAwesomeIcon data-testid={"delete-" + props.editPropertyOptions.name} className={styles.trashIcon} icon={faTrashAlt} />
-      </Button>
+      </HCButton>
     }
     <div>
-      <Button
+      <HCButton
         aria-label="property-modal-cancel"
-        size="default"
+        variant="outline-light"
         onClick={onCancel}
-      >Cancel</Button>
-      <Button
+      >Cancel</HCButton>
+      <HCButton
         aria-label="property-modal-submit"
-        form="property-form"
-        type="primary"
-        htmlType="submit"
-        size="default"
+        // form="property-form"
```

Comment:
Remove this if not necessary to keep

Reply:
this is an old property and I removed that